### PR TITLE
Cleanup and fix grammar mistakes

### PR DESCRIPTION
**Pull request by GedasGa**
### Platforms affected

electron

### Motivation and Context

Fix a few grammar mistakes and delete a typo (accidentally inserted 1).

### Testing

npm t

### Checklist

- [x] I've run the tests to see all new and existing tests pass
- [ ] I added automated test coverage as appropriate for this change
- [ ] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [x] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [x] I've updated the documentation if necessary

### CODE REVIEW

File: www/docs/en/dev/guide/platforms/electron/index.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Electron is a framework that uses web technologies (HTML, CSS, and JS) to build
 
 * **PowerShell**, for Windows 7 users, must be at version 3.0 or greater for [app signing](https://www.electron.build/code-signing#windows).
 
-* **Debugging Tools** for Windows of Windows SDK 10.0.15063.468, if you plan on creating a full distribution.
+* **Debugging Tools**, for Windows of Windows SDK 10.0.15063.468, if you plan on creating a full distribution.
```
Comment (janpio) [inline www/docs/en/dev/guide/platforms/electron/index.md]: "for Windows of Windows" doesn't sound right

File: www/docs/en/dev/guide/platforms/electron/index.md [hunk 2]
```diff
@@ -144,7 +144,7 @@ By default, the `width` is set to **800** and the `height` set to **600**. This
 #### How to make my window fullscreen?
 
 Using the `fullscreen` flag property, you can force the application to launch in fullscreen.
-1
```
Comment (janpio) [inline www/docs/en/dev/guide/platforms/electron/index.md]: this was also present in  the 9.x version of the file

Comment (GedasGa) [reply www/docs/en/dev/guide/platforms/electron/index.md]: Updated 9.x documentation too.

File: www/docs/en/dev/guide/platforms/electron/index.md (hunk 1 above)
Comment (GedasGa) [reply www/docs/en/dev/guide/platforms/electron/index.md]: Yeah, maybe that's better.

File: www/docs/en/9.x/guide/platforms/electron/index.md [hunk 3]
```diff
@@ -52,7 +52,7 @@ Electron is a framework that uses web technologies (HTML, CSS, and JS) to build
 
 * **PowerShell**, for Windows 7 users, must be at version 3.0 or greater for [app signing](https://www.electron.build/code-signing#windows).
 
-* **Debugging Tools** for Windows of Windows SDK 10.0.15063.468, if you plan on creating a full distribution.
+* **Debugging Tools for Windows**, of Windows SDK 10.0.15063.468, if you plan on creating a full distribution.
```
Comment (erisu) [inline www/docs/en/9.x/guide/platforms/electron/index.md]:
```suggestion
* **[Debugging Tools for Windows](Debugging Tools for Windows)**, for creating symbol mappings to provide better stack traceability; For the debugging toolset, it is recommended to install the Windows SDK 10.0.15063.468.
```

File: www/docs/en/dev/guide/platforms/electron/index.md [hunk 4]
```diff
@@ -52,7 +52,7 @@ Electron is a framework that uses web technologies (HTML, CSS, and JS) to build
 
 * **PowerShell**, for Windows 7 users, must be at version 3.0 or greater for [app signing](https://www.electron.build/code-signing#windows).
 
-* **Debugging Tools** for Windows of Windows SDK 10.0.15063.468, if you plan on creating a full distribution.
+* **Debugging Tools for Windows**, of Windows SDK 10.0.15063.468, if you plan on creating a full distribution.
```
Comment (erisu) [inline www/docs/en/dev/guide/platforms/electron/index.md]:
```suggestion
* **[Debugging Tools for Windows](Debugging Tools for Windows)**, for creating symbol mappings to provide better stack traceability; For the debugging toolset, it is recommended to install the Windows SDK 10.0.15063.468.
```

**Pull request merged**